src/OrchardCore.Modules/OrchardCore.Media/Assets/js/Shared/uploadComponent.js

```diff
@@ -11,18 +11,20 @@ Vue.component('upload', {
         </div> \
         ',
     props: {
-        model: Object
+        model: Object,
+        idOfUploadInputField: String
     },
     mounted: function () {
         var self = this;
-        $('#fileupload').bind('fileuploadprogress', function (e, data) {
+        var uploadInputId = self.idOfUploadInputField ?? 'fileupload';
+        $('#' + uploadInputId).bind('fileuploadprogress', function (e, data) {
```

Comment:
should we use `getElementById()` in these cases? AFAIR it works with jQuery. And doesn't need to create a new string to get the element.

Reply:
@DrewBrasher here it seems original code on left uses `$('#fileupload')` that you are trying assign dynamic id generated from server side `@mediaFieldId`

I guess using id here is unnecessory - only reason you need `@mediaFieldId` as existing code using jquery.

I would  suggest do not use jquery  here, instead just use vue's `ref` and use `this.$refs.fileupload` to refer to the element within template

Reply:
@ns8482e the element is not within the template. see https://github.com/OrchardCMS/OrchardCore/pull/12956#discussion_r1057303352

Reply:
@sebastienros I changed the event binding to use GetElementById instead of creating a new string for the jQuery selector. https://github.com/OrchardCMS/OrchardCore/pull/12956/commits/4de1b7946d393479377f3c82d2be203517b68750

Reply:
We should try as much as possible to never use JQuery so that it gets easier for us to remove JQueryUI at some point.

Reply:
@Skrypt Yes, it looks like it does use JQueryUI.